app/Http/Controllers/ForwardChatworkController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Webhook;
+use App\Models\Condition;
+use Illuminate\Http\Request;
+
+class ForwardChatworkController extends Controller
+{
+    public function forwardMessage(Request $request, $token)
+    {
+        // convert array to object with multiple level
+        $params = json_decode(json_encode((object) $request->all()), false);
+        $webhook = Webhook::where('token', $token)->first();
+
+        if ($webhook) {
+            $payloads = $webhook->payloads;
+            $filledPayloads = [];
+
+            foreach ($payloads as $payload) {
+                $conditions = $payload->conditions;
+                $isValid = true;
+
+                foreach ($conditions as $condition) {
+                    // 'return $payloads->fieldName == "value";'
+                    $expression = 'return ' . $condition->field . ' '
+                                            . $condition->operator . ' "'
+                                            . $condition->value . '";';
+                    if (!eval($expression)) {
+                        $isValid = false;
+                        break;
+                    }
+                }
+
+                if ($isValid) {
+                    array_push($filledPayloads, $payload);
+                }
+            }
+
+            // dd(collect($filledPayloads)->pluck('content'));
+
+            if ($filledPayloads) {
+                // mapping data from params request into payload content
+                $messages = [];
+                foreach ($filledPayloads as $payload) {
+                    $message = $this->generateMessage($payload->content, $params);
+                    array_push($messages, $message);
+                }
+
+                // send message to chatwork
+                dd($messages);
+            } else {
+                // make a new failed payload history record
+            }
+        } else {
+            return null;
+        }
+    }
+
+    public function generateMessage($template, $params)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Controllers/ForwardChatworkController.php</br>**Line**: 63</br>Opening parenthesis of a multi-line function call must be the last content on the line